spec/core/ics-002-client-semantics/README.md

```diff
@@ -13,53 +13,58 @@ modified: 2020-01-13
 
 ## Synopsis
 
-This standard specifies the properties that consensus algorithms of machines implementing the interblockchain
-communication protocol are required to satisfy. These properties are necessary for efficient and safe
-verification in the higher-level protocol abstractions. The algorithm utilised in IBC to verify the
-consensus transcript & state sub-components of another machine is referred to as a "validity predicate",
-and pairing it with a state that the verifier assumes to be correct forms a "light client" (often shortened to "client").
+This standard specifies the properties that consensus algorithms of state machines implementing the inter-blockchain
+communication (IBC) protocol are required to satisfy. 
+These properties are necessary for efficient and safe verification in the higher-level protocol abstractions. 
+The algorithm utilised in IBC to verify the state updates of a remote state machine is referred to as a *validity predicate*. 
```

Comment:
maybe we should consider changing this term to "state predicate", given that we use "misbehaviour predicate" below? "validity predicate" in this context is a bit redundant

Reply:
There are two predicates: a validity predicate and a misbehaviour predicate. The validity predicate is used to check the **_validity_** of incoming headers, which are then used to update the ConsensusState. Thus, I think the name is good. A "state predicate" would entail that the predicate is used to checks the state of the remote chain.